project/Dependencies.scala

```diff
@@ -75,44 +75,44 @@ object Dependencies {
   object Compile {
     // Compile
 
-    val config = "com.typesafe" % "config" % "1.4.2" // ApacheV2
-    val netty = "io.netty" % "netty" % nettyVersion // ApacheV2
+    val config = "com.typesafe" % "config" % "1.4.2"
```

Comment:
It sounds to me a better solution to this problem would be to add functionality into `sbt-license-report` to throw an error if ucategorized licenses are found (uncategorized is the fallback for when no license is detected). This can work in conjunction with `licenseFilter`.